fe/fe-core/src/main/java/org/apache/doris/alter/MaterializedViewHandler.java

```diff
@@ -1164,6 +1173,7 @@ private void onJobDone(AlterJobV2 alterJob) {
             changeTableStatus(alterJob.getDbId(), alterJob.getTableId(), OlapTableState.NORMAL);
             LOG.info("set table's state to NORMAL, table id: {}, job id: {}", alterJob.getTableId(),
                     alterJob.getJobId());
+            Env.getCurrentEnv().getGroupCommitManager().unblockTable(alterJob.getTableId());
```

Comment:
The `unblockTable` should be called before `changeTableStatus` to avoid concurrency problem